src/index.test.js

```diff
@@ -0,0 +1,15 @@
+const updateTaskStatus = require('./modules/task.js');
+
+// eslint-disable-next-line no-unused-vars
+jest.mock('./modules/task.js', () => (el, task) => el);
+
+test('Update a task after a click', () => {
+  // Set up our document body
+  const task = {
+    index: 1,
+    description: 'Read book',
+    completed: true,
+  };
+
+  expect(updateTaskStatus('Read book', task)).toBe('Read book');
```

Comment:
- [ ] I think it's better to create new tests for the clear all completed function. Check on tests if the operation is performed as expected. The knowledge of performing tests is very important to every developer's career.